src/lib/php/Application/ObligationCsvImport.php

```diff
@@ -154,20 +154,22 @@ private function handleCsvObligation($row)
     $new = $dbManager->fetchArray($resi);
     $dbManager->freeResult($resi);
 
+    $associatedLicenses = "";
+    $candidateLicenses = "";
     if (!empty($row['licnames'])) {
       $associatedLicenses = $this->AssociateWithLicenses($row['licnames'], $new['ob_pk']);
+    } else {
```

Comment:
the else case is not required since `$associatedLicenses` is initialized with that string in line 157